packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -103,6 +108,44 @@ class BuildIOSCommand extends BuildSubCommand {
       throwToolExit('Encountered error while building for $logTarget.');
     }
 
+    if (buildInfo.codeSizeDirectory != null) {
+      final SizeAnalyzer sizeAnalyzer = SizeAnalyzer(
+        fileSystem: globals.fs,
+        logger: globals.logger,
+        processUtils: processUtils,
+        appFilenamePattern: 'App'
+      );
+      // Only support 64bit iOS code size analysis.
+      final String arch = getNameForDarwinArch(DarwinArch.arm64);
```

Comment:
If so, we should make this clear somewhere in the doc

Reply:
Added to the flag documentation